docs/aspenmesh-collector.MD

```diff
@@ -4,6 +4,14 @@ The Aspen OpenTelemetry Collector collects telemetry data from Kubernetes and yo
 
 ## Installation
 
+### One-line script installation
+
+The Aspen OpenTelemetry Collector can be installed by a one-line script.
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/aspenmesh/aspenmesh-charts/main/scripts/aspenmesh-collector-setup.sh)" -- --endpoint <remote-write-endpoint> --apiKey <user-group-api-key>
```

Comment:
Unfortunately, `https://git.io/` is no longer accepting new links for shortening URL. Are we OK with the github raw content URL? Is it worth to search another free and legit third-party website?

Reply:
I think `/bin/bash` works for most users? Windows users need to either install `sh` or `bash shell` anyway